index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Benson Portfolio</title>
+</head>
+<body>
+    <header id="header">
+        <div id="headertoolbar">
+            <div id="headerlogo"><a href="#home">My Logo</a></div>
+            <nav class="navheader">
+                <button id="headermenubtn" type="button"><img src="/images/Union.png" alt="dropdown menu"></button>
```

Comment:
- [ OPTIONAL ] I recommend you check on the link to the hamburger menu. currently, on your live demo link, it is not working as expected. see the below image.
![Screenshot 2023-06-01 at 21 35 22](https://github.com/BenMKT/myPortfolio/assets/112261237/472781d2-2f4e-4610-bcd5-b27d7a4d460c)

Reply:
- Resolved bug by correcting the hamburger URL relative path.